docs/sitemap/index.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Sitemap
+layout: DetailTechnical
+sidebar:
+
+data:
+  exampleRefData: Hello from Sitemap page
+---
+
+<Labs.Sitemap href="http://localhost:3000/api/data/data" />
```

Comment:
please change the API so that its purpose is known. `api/data/data` could be anything